index.html

```diff
@@ -5838,6 +5838,26 @@ <h3>Group Control</h3>
         </p>
       </section>
     </section>
+      <section>
+      <h2>Changing the DID controller</h2>
+      <p>
+The <a>DID controller</a> for <a>DID document</a> may change over time. However,
+depending on how it is implemented, a change in the <a>DID controller</a> may
+not be apparent from changes to the <a>DID document</a> itself. For example, if
+the change is implemented through a shift in ownership of the underlying
+cryptographic keys or other controls used for one or more of the <a>verification
+methods</a> in the <a>DID document</a>, it may be indistinguishable from a
+standard key rotation.
+      </p>
+      <p>
+On the other hand, if the change is implemented by changing the value of the <code><a>controller</a></code> property, it will be transparent.
```

Comment:
```suggestion
On the other hand, if the change is implemented by changing the value of the
<code><a>controller</a></code> property, it will be transparent.
```